packages/constants/package.json

```diff
@@ -14,14 +14,13 @@
         "types": "./dist/index.d.ts",
         "default": "./dist/index.js"
       },
-      "default": "./dist/index.js",
-      "module": "./dist/index.js"
```

Comment:
Note: I yanked `module` here since `syncpack` was incorrect sorting by name, which order for `exports` does matter and so this was throwing a warning. `module` was actually redundant so no harm no foul.

Reply:
👋 Hey @brandonjpierce, 
Would you mind [creating an issue](https://github.com/JamieMason/syncpack/issues/new?assignees=&labels=&projects=&template=bug_report.md) for this please? Syncpack is _supposed_ to [sort exports](https://jamiemason.github.io/syncpack/config/sort-exports/), so this shouldn't happen. 

I tried the following but couldn't reproduce it:

```bash
git clone git@github.com:gohypergiant/standard-toolkit.git
cd standard-toolkit
git checkout 5c41d76 # the commit before this PR
syncpack format
```

If you could let me know how, I can work on a fix – thanks!